src/jdk.compiler/share/classes/com/sun/tools/javac/file/PathFileObject.java

```diff
@@ -178,13 +178,6 @@ public String inferBinaryName(Iterable<? extends Path> paths) {
             return toBinaryName(root.relativize(path));
         }
 
-        @Override @DefinedBy(Api.COMPILER)
-        public URI toUri() {
-            // Work around bug JDK-8134451:
```

Comment:
Nice to see the workaround go.  
Find the test that was added at the time of the workaround, and add this bug number (8271079) to the `@bug` line in that test, to mark that test as relevant to this issue.